Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "boringtun"
 description = "an implementation of the WireGuard® protocol designed for portability and speed"
-version = "0.3.0"
-authors = ["Vlad Krasnov <vlad@cloudflare.com>"]
```

Comment:
I see `authors` as maintainers, people actively still working the project who could respond to questions or vulnerabilities. Is this email address still even valid?

Reply:
It is not.

Reply:
IMO authors are the people to contact or email about a problem

Reply:
> IMO authors are the people to contact or email about a problem

Isn't that what issue trackers are for?

IMO the field is for announcing the property's copyright holder(likely to be @cloudflare instead in this case?) and is used for copyright-related matters(like re-license).  For product support, it should only be used as a last resort where other support contacts are unavailable.

Not really matters to me that much so feel free to disregard it.

Reply:
From [[The Manifest Format - The Cargo Book](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field)](https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field):

> The optional `authors` field lists people or organizations that are considered the "authors" of the package. The exact meaning is open to interpretation — it may list the original or primary authors, current maintainers, or owners of the package.

Oh well. :man_shrugging: